src/Search.js

```diff
@@ -11,12 +12,7 @@ const Search = () => {
           <form className="form-group search-box">
             <label htmlFor="customerName">Customer name</label>
             <div className="search-row">
-              <input
-                type="text"
-                id="customerName"
-                className="form-control"
-                placeholder="Customer name"
-              />
+              <SearchButton />
```

Comment:
This isn't quite what what exercise 1 asks you to do: it says you should move the `<button>` element to a separate component, not the `<input>`. Could you fix it?

Reply:
done